bokehjs/src/coffee/models/plots/plot_canvas.coffee

```diff
@@ -170,6 +170,12 @@ class PlotCanvasView extends Renderer.View
       # Just need to wait a small delay so container has a width
       _.delay(@resize, 10)
 
+    # Re-render when dpi changes (modern browsers emit a resize event on zoom)
+    check_resize_need_render = () ->
+      if @canvas_view.need_render()
+         @request_render()
+    $(window).on("resize", check_resize_need_render.bind(this))
```

Comment:
I don't know jquery that well, but I'll update to `$.proxy()` to be consistent :)